src/Controls/src/SourceGen/CodeBehindGenerator.cs

```diff
@@ -358,7 +358,10 @@ static void GenerateXamlCodeBehind(XamlProjectItem? xamlItem, Compilation compil
 
 		sb.AppendLine("\t\tprivate void InitializeComponent()");
 		sb.AppendLine("\t\t{");
+		sb.AppendLine("#pragma warning disable IL2026 // The body of InitializeComponent will be replaced by XamlC so LoadFromXaml will never be called in production builds");
```

Comment:
I think source suppression would probably work:
* `dotnet new maui-lib`
* Have one view or page with XAML
* `dotnet build -c Release` -> would run XamlC
* Any consuming app would have no warnings?

Reply:
Yes, XamlC will replace the body of `InitializeComponent` in that case.